File: python_sdk_asyncio/调用场景.py
import asyncio
import lebai_sdk

lebai_sdk.init()

async def main():
    # print(await lebai_sdk.discover_devices(2)) #发现同一局域网内的机器人

    robot_ip = "192.168.x.x" #设定机器人ip地址，需要根据机器人实际ip地址修改
    lebai = await lebai_sdk.connect(robot_ip, False) #创建实例
    await lebai.start_sys() #启动手臂
    await lebai.start_task("10087", None, None, False, 1) #调用场景
    await lebai.stop_sys() #停止手臂

asyncio.run(main())

File: lua_sdk/调用场景.lua
local lebai_sdk = require('lebai_sdk')
lebai_sdk.init()

function main()
    -- 设备发现
    print(lebai_sdk.discover_devices(2))

    local lebai = lebai_sdk.connect("127.0.0.1", false)
    lebai:start_sys()
    lebai:start_task("10087", nil, nil, false, 1)
end

co = coroutine.create(main)
while coroutine.status(co) ~= "dead" do
  coroutine.resume(co)
end
